bundles/org.openhab.binding.solarmax/README.md

```diff
@@ -0,0 +1,78 @@
+# SolarMax Binding
+
+This binding supports SolarMax PV inverters.
+
+## Supported Things
+
+The SolarMax MT Series is support. (tested with 8MT2 devices)
+
+## Discovery
+
+Auto-discovery is currently not available.
+
+## Thing Configuration
+
+The IP address and port number (default 12345) of the device needs to be configured.
```

Comment:
Can you add the parameters to a table list like other bindings please? Same table that the channels are listed in.... They are: host portNumber refreshInterval, note that they are the camelCase versions that the textual people will need to know.